releasenotes/README.md

```diff
@@ -155,83 +35,98 @@ Run `sfdx plugins --core` to display the version of Salesforce CLI and the insta
+
+* `force:org:shape:create`
+
+  The `definitionfile` parameter has been removed from this command. This parameter is part of the org shape feature which is currently in beta. We removed the parameter based on feedback from users that they preferred storing the org shape in the shape source org rather than in a scratch definition file.
 
-* `force:package:create --errornotificationusername (-o)`
-* `force:package:update --errornotificationusername (-o)`
+### Second Generation Packaging
 
-    Designate an active Dev Hub org user to receive email notifications for unhandled Apex exceptions, as well as install, upgrade, or uninstall failures associated with your packages. Specify the username during package creation, or when you update the package with the `force:package:update` command.
+* The `force:pacakge:version:report --verbose` command displays the release version stamped on a package version.
+
+* We've enhanced the package version CLI commands to support component deletion.
+  * If `force:package:version:create` completes successfully, and metadata has been removed from the package version as compared to its ancestor, the command displays a generic warning message about the removed metadata.
+  * The command `force:package:version:list --verbose` displays whether metadata was removed.
+  * The command `force:package:version:report` displays whether metadata was removed.
+  * The command `force:package:version:promote` displays a warning if components have been removed from the version being promoted and --noprompt isn't specified.
 
 ### Other Changes
 
-* When parsing the `.forceignore` file, Salesforce CLI uses the same rules and patterns as [git uses with the `.gitignore` file](https://git-scm.com/docs/gitignore). As a result, some entries in your existing `.forceignore` files might be invalid. The `force:source:*` commands warn you about any invalid entries.  
+* FIX: We've improved the error message returned when you run `force:user:password:generate` using API version 51.0 and EnableSetPasswordInApi is configured as a security setting in your scratch org definition file. EnableSetPasswordInApi is now a scratch org feature instead of a Metadata API setting. Here's an example of configuring it as a feature in your scratch org definition file:
+
+  `"features": ["EnableSetPasswordInApi","MultiCurrency"],`
+
+  This change is a result of the field `Settings.securitySettings.passwordPolicies.enableSetPasswordInApi` being [removed in version 51.0 of the Metadata API](https://help.salesforce.com/articleView?id=release-notes.rn_api_meta.htm&type=5&release=230).  (GitHub issue [#798](https://github.com/forcedotcom/cli/issues/798))
 
-* FIX: The `force:source:convert` command works correctly when you individually convert two or more custom fields of the same custom object.  
+* FIX: The `force:user:password:generate` no longer fails when generating a password for a user that does not have access to Profile Standard Object.
```

Comment:
Edit: The `force:user:password:generate` command no longer failes when generating a password for a user that doesn't have access to the Profile standard object." 

Is there an associated GitHub issue?  We usually only list bug fixes that have an accompanying github issue.  If so, please add here with link.  See previous FIX.